### PR TITLE
Make time input simple input box with validation

### PR DESCRIPTION
**Pull request by Peartes**
As discussed in our sync @dkent600 @renc-curvelabs @BartuCulha the time input found.  on the 4th stage of seed, registration has been a simple input box with a placeholder describing the format of the time and validation in place to catch wring usage. The issue referred to is [here](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=09a5d393720747aeb0c3a8b5bf51ca85)

### CODE REVIEW

**Review by dkent600 — CHANGES_REQUESTED** [review]
Validation should confirm that the hours are between 0 and 23 and minutes are between 0 and 59

File: src/newSeed/stage4.ts [hunk 1]
```diff
@@ -71,15 +71,21 @@ export class Stage4 extends BaseStage {
       message = "Please select a Start Date";
     } else if (!this.startTime) {
       message = "Please enter a value for the Start Time";
-    } else if (!(Number.parseInt(this.startTime.split(":")[0]) < 25)
-      || !(Number.parseInt(this.startTime.split(":")[1]) < 61)) {
+    } else if (!(Number.parseInt(this.startTime.split(":")[0]) >= 0)
+      || !(Number.parseInt(this.startTime.split(":")[0]) < 24)) {
+      message = "Please enter a valid value for Start Time";
+    } else if (!(Number.parseInt(this.startTime.split(":")[1]) >= 0)
+      || !(Number.parseInt(this.startTime.split(":")[1]) < 60)) {
```
Comment (dkent600) [inline src/newSeed/stage4.ts:78]: what if they enter a negative number for hours or minutes?

File: src/newSeed/stage4.ts [hunk 2]
```diff
@@ -71,15 +71,21 @@ export class Stage4 extends BaseStage {
       message = "Please select a Start Date";
     } else if (!this.startTime) {
       message = "Please enter a value for the Start Time";
-    } else if (!(Number.parseInt(this.startTime.split(":")[0]) < 25)
-      || !(Number.parseInt(this.startTime.split(":")[1]) < 61)) {
+    } else if (!(Number.parseInt(this.startTime.split(":")[0]) >= 0)
```
Comment (dkent600) [inline src/newSeed/stage4.ts:74]: you should only execute each split once.

Comment (dkent600) [inline src/newSeed/stage4.ts:74]: what happens if there is no ':'?

or what happens if the value is not a number (parseInt returns NaN)

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/stage4.ts (hunk 1 above)
Comment (Peartes) [reply src/newSeed/stage4.ts:78]: Validation would flag it

File: src/newSeed/stage4.ts (hunk 2 above)
Comment (Peartes) [reply src/newSeed/stage4.ts:74]: Validation flags it

Comment (Peartes) [reply src/newSeed/stage4.ts:74]: Validation catches all corner cases I tested but please in case you think of any other

Comment (Peartes) [reply src/newSeed/stage4.ts]: Done

**Review by dkent600 — APPROVED** [review]

**Pull request merged**